hack/test-stirunimage.sh

```diff
@@ -86,6 +86,20 @@ test_debug "s2i build with relative path with file://"
 s2i build file://./cakephp-ex openshift/php-55-centos7 test --loglevel=5 &> "${WORK_DIR}/s2i-rel-proto.log"
 check_result $? "${WORK_DIR}/s2i-rel-proto.log"
 
+if [[ "$OSTYPE" == "cygwin" ]]; then
+   test_debug "s2i build with windows volume options"
+   s2i build cakephp-ex openshift/php-55-centos7 test --volume "${WORK_DIR}:/home/:z"
+   check_result $? "${WORK_DIR}/s2i-volume-correct.log"
+   ! s2i build cakephp-ex openshift/php-55-centos7 test --volume "${WORK_DIR}:/home/"
```

Comment:
I don't love the leading `!` - perhaps check_result could be extended to deal with expected negatives?